src/components/Patient/PatientHome.tsx

```diff
@@ -511,32 +534,30 @@ export const PatientHome = (props: {
                           <CareIcon icon="l-file-upload" className="text-xl" />
                           {t("view_update_patient_files")}
                         </span>
-                      </ButtonV2>
+                      </Button>
                     </div>
 
                     {NonReadOnlyUsers && (
```

Comment:
Although was there before, `NonReadOnlyUsers` is a function and boolean check would always return true right?